.gitignore

```diff
@@ -45,6 +45,11 @@ coverage.xml
 # Pycharm
```

Comment:
@elicoidal @pedrobaeza 

?  I just got asked to not remove this.  Please decide  between you whether to remove or not.